### DISCUSSION TITLE
[Bug] Collapsible Markdown behave differently between Mobile App and Mobile WebApp

### DISCUSSION THREAD

**Discussion by ranggakd** [community/community#34809] [Mobile] [answered]
## This bug compares these apps:
- [x] GitHub Website App (open in a mobile browser)
- [x] Github Mobile (Android version)

## Which apps are you using and what versions of it:
Mobile Browser: Google Chrome 105.5195.136
Android 12: GitHub 1.82.0

## The steps to reproduce this issue:
1. Open a GitHub repository that contains a collapsible markdown that has heading in it like [this](https://github.com/ranggakd/List-All-Programming-Telegram-Group) on both website app and mobile app.
2. The view results are different between those apps although they are accessed from the mobile.

## Description of the bug
The view results of collapsible markdown that has heading in it don't appear the same across platform. This view below from the mobile app.

![image](https://user-images.githubusercontent.com/36787606/193415992-0ea17b80-94d0-40f0-9a64-d8740c70cec1.png)

## Expected behaviour
The view results of collapsible markdown that has heading in it appear the same across platform. This view below from the website app accessed from the mobile browser.

![image](https://user-images.githubusercontent.com/36787606/193416024-66a9fa41-6d2e-473e-8832-ffe14e3d30dd.png)

## Snippet Code
```markdown
<details open>
<summary>
<!-- empty line -->
## Heading 2
</summary>
<!-- empty line -->
  - [Unorder links](#heading_referenced)
  - [Unorder links](#heading_referenced)
</details>
<!-- empty line -->
```

[Source 1](https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab)
[Source 2](https://gist.github.com/scmx/eca72d44afee0113ceb0349dd54a84a2)

**Comment by HenSquared** [comment, marked as answer]
Hey @ranggakd. I'm able to reproduce this issue on both iOS and Android. I'll open an issue so the team can investigate!
Thanks for your help.

**Reply by ranggakd** [reply]
Thank you much appreciated @HenSquared
